### PR TITLE
[UI] Cleanup & fixes

### PR DESCRIPTION
**Pull request by iverc**
## 📃 Summary

- [X] Remove X icon
- [X] Add liquidity pool info urls
- [X] Cleanup unused code

## 🎯 Asana Task
https://app.asana.com/0/1196233999049111/1199518867537113

### CODE REVIEW

File: ui/app/src/components/poolList/SinglePool.vue [hunk 1]
```diff
@@ -153,7 +151,13 @@ export default defineComponent({
           </div>
         </div>
       </div>
-
+      <div class="section">
+        <div class="info">
+          <h3 class="mb-2">Liquidity provider rewards</h3>
+          <p class="text--small mb-2">Liquidity providers earn a 0.3% fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.</p>
```
Comment (59023g) [inline ui/app/src/components/poolList/SinglePool.vue:157]: [Per Casey](https://sifchain.slack.com/archives/G01BX71QY1F/p1610477848293900?thread_ts=1610471699.290600&cid=G01BX71QY1F), change copy and link in `<p>` below to: 
"Liquidity providers earn a percentage fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity. To learn more, reference of documentation <a href="https://app.gitbook.com/@sifchain/s/sifchain/roles/liquidity-providers">here</a>."

File: ui/app/src/views/PoolPage.vue [hunk 2]
```diff
@@ -51,31 +50,20 @@ export default defineComponent({
...
-        </div> -->
-      <!-- </PriceCalculation> -->
-      <PoolList class="mb-2">
-        <PoolListItem
-          v-for="(accountPool, index) in accountPools"
-          :key="index"
-          :accountPool="accountPool"
-          @click="selectedPool = accountPool"
-        />
+          <p class="text--small mb-2">Liquidity providers earn a 0.3% fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.</p>
```
Comment (59023g) [inline ui/app/src/views/PoolPage.vue:56]: Same here with ^^ copy

**Review by 59023g — DISMISSED** [review]
Looks good just change the text

**Comment by ryardley** [comment]
>  Cleanup unused code 

🙏

**Review by ryardley — DISMISSED** [review]
🎸

**Comment by iverc** [comment]
@ryardley please approve again, just fixed merge conflicts.

**Review by ryardley — APPROVED** [review]

**Pull request merged**